src/lib.rs

```diff
@@ -398,7 +394,7 @@ pub(crate) fn rav1d_open(s: &Rav1dSettings) -> Rav1dResult<Arc<Rav1dContext>> {
         allocator: s.allocator.clone(),
         logger: s.logger.clone(),
         apply_grain: s.apply_grain,
-        operating_point: s.operating_point,
+        operating_point: s.operating_point.get(),
```

Comment:
```suggestion
        operating_point: s.operating_point.get(), // TODO propagate `InRange`
```

You don't have to fully propagate this now, but we should at least leave a TODO for it.